### PR TITLE
Added file to testing on video and updated requirements.txt

### PR DESCRIPTION
**Pull request by levindabhi**
With test_video.py on can style transfer on content video with style image/video. 
Added conditions to verify length of content and style video is same if one is testing with style video. 
Updated requirements.txt with opencv-python and imageio used in test_video.py.

### CODE REVIEW

File: test_video.py [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+content_video = cv2.VideoCapture(args.content_video)
+fps = int(content_video.get(cv2.CAP_PROP_FPS))
+content_video_length = int(content_video.get(cv2.CAP_PROP_FRAME_COUNT))
+output_width = int(content_video.get(cv2.CAP_PROP_FRAME_WIDTH))
+output_height = int(content_video.get(cv2.CAP_PROP_FRAME_HEIGHT))
+
+assert fps!=0, 'Fps is zero, Please enter proper video path'
+
+pbar = tqdm(total = content_video_length)
+if(str(style_path).endswith('.mp4') or str(style_path).endswith('.mpg') or str(style_path).endswith('.avi')):
```
Comment (naoto0804) [inline test_video.py:127]:
```
if style_path.suffix in [".mp4", ".mpg", ".avi"]:
```
would be more simple

File: test_video.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+style_tf = test_transform(args.style_size, args.crop)
+        
+#get video fps&video size
+content_video = cv2.VideoCapture(args.content_video)
+fps = int(content_video.get(cv2.CAP_PROP_FPS))
+content_video_length = int(content_video.get(cv2.CAP_PROP_FRAME_COUNT))
+output_width = int(content_video.get(cv2.CAP_PROP_FRAME_WIDTH))
+output_height = int(content_video.get(cv2.CAP_PROP_FRAME_HEIGHT))
+
+assert fps!=0, 'Fps is zero, Please enter proper video path'
```
Comment (naoto0804) [inline test_video.py:124]: Could you consider using the code formatter for Python such as [autopep8](https://pypi.org/project/autopep8/0.8.2/) for maintaining consistency with my codes?
I think whitespace should be placed before and after arithmetic operators.
(And this applies to the whole code)

File: test_video.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        #output = np.uint8(output)
+        output = np.transpose(output, (1,2,0))
+        output = cv2.resize(output, (output_width, output_height), interpolation=cv2.INTER_CUBIC)
+
+        writer.append_data(np.array(output))
+        pbar.update(1)
+    
+    content_video.release()
+else:
+    assert True, 'Please enter content and style video with proper video extenstion'
```
Comment (naoto0804) [inline test_video.py:206]: This comment will never show up;

**Review by naoto0804 — COMMENTED** [review]
Thank you for your PR! Although providing the code to do style transfer on videos is nice for this repo. I cannot merge this PR in the current form since there are some coding style inconsistencies. I would appreciate it if you could reflect my comments!

**Comment by levindabhi** [comment]
Thank you for reviewing my code. Based on your comments, I have updated test_video.py. 
Please let me know if any changes needed.

**Comment by naoto0804** [comment]
thanks!

**Pull request merged**